modules/cnv-importing-vmware-vm.adoc

```diff
@@ -1,67 +1,77 @@
 
-* If you select a saved vCenter instance, the wizard connects to it by using the saved credentials.
+.. Select the network interface model.
+.. Select the network definition.
+.. Select the network interface type.
+.. Enter the MAC address.
+.. Click *Save* and then click *Next*.
 
-.. Click *VM to import* and select a virtual machine or template.
+. In the *Storage* screen, perform the following steps:
+
+.. Click the Options menu {kebab} of a disk and select *Edit*.
++
+.. Enter a valid name.
 +
-The *Name* and *Operating System* are filled in automatically with the attributes of the selected virtual machine or template.
+The name can contain lower-case letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain upper-case letters, spaces, periods (`.`), or special characters.
 
-.. Specify the *Memory (GB)* and number of *CPUs* if you use the default flavor, *Custom*.
+.. Select the interface type.
+.. Select the storage class.
 +
-Optionally, you can select an existing *Flavor*.
+If you do not select a storage class, {CNVProductName} uses the default storage class to create the virtual machine.
 
-.. Select a *Workload Profile*.
-.. Click *Next*.
-. Perform the following procedure in the *Networking* screen:
-.. Click *NETWORK CONFIGURATION* and select *Pod Networking*.
-.. Select a *BINDING METHOD*.
-.. Click the &#10003; button to save the update.
-.. Click *Next*.
-. Perform the following procedure in the *Storage* screen:
-.. Click each disk and select a *STORAGE CLASS*. If you do not select a storage class, {CNVProductName} uses the default storage class to create the virtual machine.
+.. Click *Save* and then click *Next*.
+
+. In the *Advanced* screen, enter the *Hostname* and *Authorized SSH Keys* if you are using `cloud-init`.
+. Click *Next*.
 
-.. Click the &#10003; button to save the update.
-.. Select a *Bootable Disk* if you have more than one bootable disk.
-.. Click *Create Virtual Machine >*.
+. Review your settings and click *Create Virtual Machine*.
 +
-The *Result* screen displays the resources created for the virtual machine.
-. Click *Close*.
+A *Successfully created virtual machine* message and a list of resources created for the virtual machine are displayed. The powered off virtual machine appears in *Workloads* -> *Virtual Machines*.
 
-To view the progress of the import:
+. Click *See virtual machine details* to view the dashboard of the imported virtual machine.
 
-. Click *Workloads* -> *Pods* from the side menu.
-. Click the conversion pod, *kubevirt-v2v-conversion-_<id>_*, to view the pod details.
-. In the *Overview* tab, click *ANNOTATIONS* for the *v2vConversionProgress* value.
+.Troubleshooting
```

Comment:
This is interesting. I don't think it's part of the mod docs templates, but I think it's not a bad idea to consider.

Reply:
Troubleshooting is part of minimalist principles. But I decided to eliminate the header and add the steps as a subprocedure to the final step.

Reply:
Right, but the heading isn't part of the templates. But I'd be curious to hear wither anyone thinks this would be an interesting add. I think it's kinda nice actually pointing it out like that. But yeah, most people I assume do it add it to the last step or in a verification step.